tests/apps/pubsub-bulk-subscriber/app.go

```diff
@@ -0,0 +1,577 @@
+/*
+Copyright 2021 The Dapr Authors
```

Comment:
THis app seems very similar to `pubsub-subscriber`. Any chance they could be made into the same app (just like it was done for gRPC)? At a high level, the less apps, the better (each app takes 1 min to be deployed on an AKS test cluster)

Reply:
Actually, I had started by having my topics and all other bulk related methods in `pubsub-subscriber` app only. I am using kafka for bulk subscribe, while this App has been using redis. So, this app was now using both redis and kafka. I spent around 2 days in stabilizing it. It used to pass sometimes and sometime fail - it was quite flaky. Debugging and through logging, I realized that this App was not working properly when subscribing items from redis and from kafka pubsub as well - but only for http calls. I tried to play with locks and all, but it all didn't seem to come to a logical conclusion. 
When I tried same code with different App - it was working fine. So, now I have topic which I bulk-subscribe and also topic which I normal subscribe, and it is all green.

Whereas, in grpc, it didn't seem like an issue.
I parked this for later to check that why it is an issue in case of http. It may be the way App has been setup or something particularly related to http. But, for this PR, I would like to keep it in a separate App only.

Reply:
Ok, makes sense